app/views/home/index.html.erb

```diff
@@ -1,35 +1,185 @@
 <header class="jumbotron text-center bg-gray-light mb-6 border-bottom px-3">
   <div class="container-lg position-relative">
     <h1 class="alt-h1">Open Source Friday</h1>
-    <p class="alt-lead text-gray mb-md-5 col-md-8 mx-auto"><%= @subtitle || "Contribute to open source every Friday, and help new contributors, maintainers, and companies do the same." %></p>
+    <p class="alt-lead text-gray mb-md-5 col-md-8 mx-auto"><%= @subtitle || "Contribute to open source every Friday, and help new contributors, maintainers,
+and companies do the same." %></p>
   </div>
 </header>
 
-<div class="container-lg my-6 px-3">
-  <p class="lead my-6">Open Source Friday is a movement to encourage people to contribute to open source every Friday. We commit as contributors to submit at least one contribution on an existing open source project, as maintainers to review and merge incoming contributions and as companies to encourage and help our managers, coworkers and reports to make a contribution.</p>
+<div class="container-md py-6 py-3 px-3">
+  <h2 class="alt-h2 text-purple text-center">Your business depends on it</h2>
```

Comment:
How about `organisation` (or even `organization` 😉) so we include non-profits etc.?  If you think it reduces the impact: feel free to 👎